Equipo2Hosteleria/src/main/java/Vista/PanelListaCompra.java

```diff
@@ -0,0 +1,139 @@
+		add(lblTicketFactura);
+		
+		btnDesconectarse = new JButton("Desconectarse");
+		btnDesconectarse.setBounds(137, 235, 146, 23);
+		add(btnDesconectarse);
+		
+		btnVolver = new JButton("Volver");
+		btnVolver.setBounds(38, 235, 89, 23);
+		add(btnVolver);
+		
+		lblProductos = new JLabel("Productos:");
+		lblProductos.setBounds(38, 36, 89, 19);
+		add(lblProductos);
+		
+		grupoBotones = new ButtonGroup();
+		
+		lblTotal = new JLabel("Total:");
+		lblTotal.setBounds(306, 207, 45, 19);
+		add(lblTotal);
+		
+		scrollPane = new JScrollPane();
+		scrollPane.setBounds(38, 61, 368, 135);
+		add(scrollPane);
+		
+		textPane_Productos = new JTextPane();
+		textPane_Productos.setEditable(false);
+		scrollPane.setViewportView(textPane_Productos);
+		textPane_Productos.setText(ListaCompra);
+		
+		textPane_Total = new JTextPane();
+		textPane_Total.setEditable(false);
+		textPane_Total.setBounds(343, 207, 63, 20);
+		add(textPane_Total);
+		textPane_Total.setText(""+ListaCompraTotal);
+		
+		initializeEvents();
+	}
+	
+	private void initializeEvents() {
+		this.btnVolver.addActionListener(listenerBotonVolver(this.controladorPanelListaCompra));
+		this.btnDesconectarse.addActionListener(listenerBotonDesconectarse(this.controladorPanelListaCompra));
+		this.btnBorrarLista.addActionListener(listenerBotonBorrarLista(this.controladorPanelListaCompra));
+	}
+	
+	
+	private ActionListener listenerBotonVolver(ControladorPanelListaCompra controladorPanelListaCompra) {
+		return new ActionListener() {
+			public void actionPerformed(ActionEvent arg0) {
+				System.out.println("Ejecutando evento Boton Volver");
+				controladorPanelListaCompra.accionadoBottonVolverPanelListaCompra();
+			}
+		};
+	}
+	
+	private ActionListener listenerBotonDesconectarse(ControladorPanelListaCompra controladorPanelListaCompra) {
+		return new ActionListener() {
+			public void actionPerformed(ActionEvent arg0) {
+				System.out.println("Ejecutando evento Boton Desconectarse");
+				controladorPanelListaCompra.accionadoBottonDesconectarsePanelListaCompra();
+			}
+		};
+	}
+	
+	private ActionListener listenerBotonBorrarLista(ControladorPanelListaCompra controladorPanelListaCompra) {
+		return new ActionListener() {
+			public void actionPerformed(ActionEvent arg0) {
+				System.out.println("Ejecutando evento Boton Borrar");
+				controladorPanelListaCompra.accionadoBottonBorrarListaPanelListaCompra();
+				
+				ListaCompra = controladorPanelListaCompra.obtenerListaCompra();
+				ListaCompraTotal = ""+controladorPanelListaCompra.obtenerTotalCarro();
+				
+				textPane_Productos = new JTextPane();
+				textPane_Productos.setEditable(false);
+				scrollPane.setViewportView(textPane_Productos);
+				textPane_Productos.setText(ListaCompra);
+				
+				textPane_Total = new JTextPane();
+				textPane_Total.setEditable(false);
+				textPane_Total.setBounds(343, 207, 63, 20);
+				add(textPane_Total);
+				textPane_Total.setText(""+ListaCompraTotal);
```

Comment:
Si vais a crear nuevos paneles, porque no lo hacéis como el con el resto y navegáis mediante el controlador? Es decir, porque no hacéis para cada nuevo panel un controlador?